docs/Troubleshooting.md

```diff
@@ -26,13 +26,13 @@ On Windows you can find the process using port 8081 using [Resource Monitor](htt
 
 #### Using a port other than 8081
 
-You can configure the packager to use a port other than 8081 by using the `port` parameter:
+You can configure the packager to use a port other than 8081 by using the `port` parameter.  You should start this before performing the build:
 
 ```
 $ react-native start --port=8088
 ```
 
-You will also need to update your applications to load the JavaScript bundle from the new port. Open the in-app [Developer menu](docs/debugging.html#accessing-the-in-app-developer-menu), then go to **Dev Settings** → **Debug server host for device** and replace 8081 with your port of choice.
+You will also need to update your applications to load the JavaScript bundle from the new port. Perform a search and replace in the iOS or android code to change port 8081 to 8088 (or whichever port you decided to use).
```

Comment:
If port 8081 is in use when you try to launch the app (for example, react-native run-ios), the app will fail to launch with the message "Port 8081 already in use, packager is either not running or not running correctly". Since the react-native app doesn't start on the emulator, there is no way to bring up the app settings menu. The app does appear to start with Android, so maybe this is just an issue that affects the iOS flavor.